semgrep-core/bin/Main.ml

```diff
@@ -878,7 +882,14 @@ let main () =
   in
 
   (* does side effect on many global flags *)
-  let args = Common.parse_options (options()) usage_msg (Array.of_list argv) in
+  let _args_tmp = Common.parse_options (options()) usage_msg (Array.of_list argv) in
```

Comment:
Do not use _ as the first chat. It is used to silence warnings about unused variable. Here the var will be used